web/template/watch.gohtml

```diff
@@ -6,6 +6,7 @@
     {{$stream := .IndexData.TUMLiveContext.Stream}}
     {{$course := .IndexData.TUMLiveContext.Course}}
     <title>{{.IndexData.Branding.Title}} | {{$course.Name}}: {{$stream.Name}}</title>
+    <link rel="canonical" href="{{.IndexData.CanonicalURL.Stream $course.Slug $stream.ID}}" />
```

Comment:
This needs to take the version into account (e.g. CAM)

Reply:
great catch. thanks! 👍